steps/pagerduty/0.1.1/step.yml

```diff
@@ -0,0 +1,98 @@
+title: PagerDuty
+summary: |
+  Create a PagerDuty alert
+description: |
+  This step creates a PagerDuty alert with specific parameters. It can use Events API v1 or v2.
+website: https://github.com/PagerDuty/bitrise-step-pagerduty
+source_code_url: https://github.com/PagerDuty/bitrise-step-pagerduty
+support_url: https://github.com/PagerDuty/bitrise-step-pagerduty/issues
+published_at: 2019-10-11T14:29:40.326045-04:00
+source:
+  git: https://github.com/PagerDuty/bitrise-step-pagerduty.git
+  commit: b360e8c9252741cccc212b026be9499e70be0c11
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
```

Comment:
toolkit can optionally be omited as bash with step.sh is the default option used in this case.